pds_github_util/release/maven_release.py

```diff
@@ -13,14 +13,17 @@ def maven_get_version(workspace=None):
     pom_doc = etree.parse(pom_path)
     r = pom_doc.xpath('/pom:project/pom:version',
                       namespaces={'pom': 'http://maven.apache.org/POM/4.0.0'})
-    return r[0].text
+    version = r[0].text
+    print(f'Yo yo maven gets version {version} from the pom', file=sys.stderr)
```

Comment:
enjoying this stderr print statements

Reply:
@jordanpadams I'm always happy to see things written to `stderr` 😊 

(Seriously, a remnant left from before @MJJoyce and I cleaned up the mis-configured logging all over the code. I vote to leave it—sort of like that bit of hay you find in grandma's hand-knit sweater.)